src/Umbraco.Core/Models/PublishedContent/PublishedPropertyType.cs

```diff
@@ -23,6 +23,7 @@ public class PublishedPropertyType : IPublishedPropertyType
         private PropertyCacheLevel _deliveryApiCacheLevel;
 
         private Type? _modelClrType;
+        private Type? _modelDeliveryApiClrType;
```

Comment:
To remain consistent with the current field naming, this should be named `_deliveryApiModelClrType`, just as we have `_deliveryApiCacheLevel` as the Delivery API equivalent for `_cacheLevel`